autogpt_platform/backend/backend/util/exceptions.py

```diff
@@ -4,3 +4,18 @@ class MissingConfigError(Exception):
 
 class NeedConfirmation(Exception):
     """The user must explicitly confirm that they want to proceed"""
+
+
+class InsufficientBalanceError(ValueError):
+    user_id: str
+    message: str
+    balance: float
+    amount: float
+
+    def __init__(self, message: str, user_id: str, balance: float, amount: float):
+        super().__init__(message)
+        self.args = (message, user_id, balance, amount)
+        self.message = message
+        self.user_id = user_id
+        self.balance = balance
+        self.amount = amount
```

Comment:
What does it look like on the block error output?
Does it only show the message only?
What about when printed as an exception?

Reply:
Looks like this ('Insufficient balance of $0.01, where this will cost $0.04', '8c18fc64-d4d3-42d9-9f10-c4d2f4657f9a', 1, -4)
![image](https://github.com/user-attachments/assets/6c734b83-a14c-4486-8054-e72e8d70623b)

Reply:
![image](https://github.com/user-attachments/assets/4466b628-93e2-4214-8e07-d9f9ddddbe91)
fixed it to this